src/nanoemoji/config.py

```diff
@@ -288,17 +303,19 @@ def write(dest: Path, config: FontConfig):
             for m in config.masters
         },
     }
-    dest.write_text(toml.dumps(toml_cfg))
+    with open(dest, "w") as toml_file:
+        tomlkit.dump(toml_strip(toml_cfg), toml_file)
 
 
 def _resolve_config(
     config_file: Optional[Path] = None,
 ) -> Tuple[Optional[Path], MutableMapping[str, Any]]:
     if config_file is None:
-        with resources.path("nanoemoji.data", _DEFAULT_CONFIG_FILE) as config_file:
-            # no config_dir in this context; bad input if we need it
-            return None, toml.load(config_file)
-    return config_file.parent, toml.load(config_file)
+        # no config_dir in this context; bad input if we need it
+        toml_cfg = resources.files(nanoemoji.data).joinpath(_DEFAULT_CONFIG_FILE).read_text()
```

Comment:
I see `importlib.resources.path` is now deprecated and they want us to use `filez` instead, that's ok. Does't that also support passing the package name as a string? "nanoemoji.data" instead of `import nanoemoji.data` that you did above? I would prefer the string if possible, thanks

Reply:
Fixed